docs/pages/blog/premium-plan-release.md

```diff
@@ -0,0 +1,140 @@
+We’re happy to announce that the Premium plan is [finally out](https://mui.com/pricing/)!
+With it, MUI X officially steps up to the next level, supporting the most advanced use cases for UI components.
+
+The new Premium features are intended to support end-users with the most sophisticated tools to navigate and draw insights from vast amounts of data.
+
+## Row grouping
+
+We started with [row grouping](https://mui.com/x/react-data-grid/row-grouping/), which enables users to group data based on repeating values in the grid.
+For example, users can now group orders by buyers, movies by directors, or cities by the governing parties.
+
+These kinds of use cases would usually be dealt with using a different query on the database, perhaps requiring a new service end-point, maybe even a new front-end, with all the UX challenges this would entail.
+
+Now it’s available at the end-user's fingertip, directly in the data grid.
+Users are encouraged to explore the data and create combinations with multiple grouping criteria,
+and are then able to export these new views to Excel, with the latest Premium feature: Excel export.
+
+## Excel export
+
+One of the most requested features,
+the [“Excel export”](https://mui.com/x/react-data-grid/export/#excel-export) enables users to bring the data from the grid into Excel, while mirroring the existing visualization in the data grid based on filters, sorting, and row grouping.
+
+In the future we intend to support all of the features impacting visualization of the data grid, but we’d love to hear from you if there’s anything in particular you’d like to see in future releases.
+
+## Aggregation (Coming in late Q2)
+
+Speaking of future releases, [aggregation functions](https://mui.com/x/react-data-grid/aggregation/#) are coming next, in a further step to improving tools to analyze data.
+Combined with row grouping, the aggregation functions help users dive deeper to get data insights.
+
+Expanding on the use cases for the previous examples, users will be able to dynamically get the sum of orders for each user, the minimum rating of each director, or the average crime rate for each governing party.
+
+That and much more just a few clicks away.
+
+# The new licensing model
+
+As we release the Premium plan, we’d also like to announce its new price and MUI X licensing model.
+
+We’re going for simplicity, restructuring our model to be as clear as possible, based on the recurrent questions we have had on our sales and other channels.
+We’d also like to keep it viable for individual and small teams, while still capturing more of the value we generate for larger companies.
+
+Here is a list of everything that has changed:
+
+> ⚠️ Please note: The following changes apply only to **new customers**. Existing customers are not impacted, and their license agreement remains valid.
+
+- **Price points**
+
+  Balancing our goals and pricing structural changes, we’re lowering our base price points.
+
+  - Pro: $249 → $180 / year
```

Comment:
```suggestion
  - Pro: $249 → $180/year
```
@samuelsycamore?

Should we show these as monthly prices to be consistent with the store, or is it confusing in this context?

```suggestion
  - Pro: $249/dev → $15/dev/month, billed annually
```

Reply:
I considered adding this, but I believe it's too much information at once. 
Maybe we can bring back the section dedicated to the monthly display.

Reply:
@mbrookes.
I've added a second list to display the prices per month.